docs/best-practices/mocking-request.md

```diff
@@ -101,4 +101,36 @@ const badRequest = await bankTransfer('1234567890', '100')
 // subsequent request to origin http://localhost:3000 was not allowed (net.connect disabled)
 ```
 
+## Reply with data based on request
 
+If the mocked response needs to be dinamically derived from the reuqest parametes, you can provide a function instead of an object to `reply`
```

Comment:
```suggestion
If the mocked response needs to be dynamically derived from the request parameters, you can provide a function instead of an object to `reply`
```